lib/v2/screens/explore_screens/explore/explore_screen.dart

```diff
@@ -94,29 +93,37 @@ class ExploreScreen extends StatelessWidget {
                               amountLabel: 'Trust Tokens Remaining'.i18n,
                             ),
                           ),
+                          const SizedBox(width: 20),
+                          const Expanded(child: SizedBox.shrink()),
                         ],
                       ),
                     ),
-                    const SizedBox(height: 20),
+                    const SizedBox(height: 100),
                     Padding(
-                      padding: const EdgeInsets.only(left: 16, right: 16),
+                      padding: const EdgeInsets.only(left: 16, right: 16, bottom: 16),
                       child: Row(
-                        mainAxisSize: MainAxisSize.max,
                         children: [
                           Expanded(
-                            child: ExploreInfoCard(
-                              onTap: () => launch('$buySeedsUrl${settingsStorage.accountName}', forceSafariVC: false),
+                            child: ExploreLinkCard(
                               title: 'Get Seeds'.i18n,
-                              icon: SvgPicture.asset('assets/images/explore/link_chain.svg'),
+                              backgroundImage: 'assets/images/explore/card_get_seeds.jpg',
+                              logoImage: 'assets/images/explore/get_seeds_logo.svg',
+                              gradient: const RadialGradient(
+                                center: Alignment(-0.6, 0.6),
+                                radius: 3,
+                                colors: [Color.fromARGB(255, 47, 241, 65), Color.fromARGB(255, 43, 72, 43)],
+                              ),
+                              onTap: () => launch('$buySeedsUrl${settingsStorage.accountName}', forceSafariVC: false),
                             ),
                           ),
-                          const SizedBox(width: 20),
+                          const SizedBox(width: 16),
                           Expanded(
                             child: (state.isDHOMember != null && state.isDHOMember == true)
-                                ? ExploreInfoCard(
+                                ? ExploreLinkCard(
+                                    title: 'Hypah DHO',
+                                    backgroundImage: 'assets/images/explore/card_hypha.jpg',
+                                    logoImage: 'assets/images/explore/hypha_dho_logo.svg',
                                     onTap: () => NavigationService.of(context).navigateTo(Routes.dho),
-                                    title: 'Hypha DHO',
-                                    icon: SvgPicture.asset('assets/images/explore/link_chain.svg'),
                                   )
```

Comment:
These are need conplex UI work so I'll leave it to @gguij004 to see if he can find a better solution.